src/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -1044,10 +1069,12 @@ public int CompareTo(object obj)
         /// return an array of one byte whose element is 0x00.
         /// </summary>
         /// <returns></returns>
-        public byte[] ToByteArray()
+        public byte[] ToByteArray() => ToByteArray(false, false);
```

Comment:
Nit: can you name the args, e.g. `ToByteArray(isUnsigned: false, isBigEndian: false)`?